Packages/com.unity.inputsystem/InputSystem/Plugins/XR/Haptics/GetHapticCapabilitiesCommand.cs

```diff
@@ -35,13 +94,22 @@ public struct GetHapticCapabilitiesCommand : IInputDeviceCommandInfo
         [FieldOffset(InputDeviceCommand.kBaseCommandSize)]
         public uint numChannels;
 
-        [FieldOffset(InputDeviceCommand.kBaseCommandSize + sizeof(uint))]
+        [FieldOffset(InputDeviceCommand.kBaseCommandSize + 4)]
+        public bool supportsImpulse;
+
+        [FieldOffset(InputDeviceCommand.kBaseCommandSize + 5)]
+        public bool supportsBuffer;
+
+        [FieldOffset(InputDeviceCommand.kBaseCommandSize + 8)]
         public uint frequencyHz;
 
-        [FieldOffset(InputDeviceCommand.kBaseCommandSize + (sizeof(uint) * 2))]
+        [FieldOffset(InputDeviceCommand.kBaseCommandSize + 12)]
         public uint maxBufferSize;
 
-        public HapticCapabilities capabilities => new HapticCapabilities(numChannels, frequencyHz, maxBufferSize);
+        [FieldOffset(InputDeviceCommand.kBaseCommandSize + 16)]
+        public uint optimalBufferSize;
+
+        public HapticCapabilities capabilities => new HapticCapabilities(numChannels, supportsImpulse, supportsBuffer, frequencyHz, maxBufferSize, optimalBufferSize);
```

Comment:
Maybe an xmldoc for the new property? (I know the rest of the struct is already missing docs)

Reply:
I added docs to everything.